src/components/Dashboard/Category-Card/AdminModal.js

```diff
@@ -0,0 +1,193 @@
+import React, { useState } from "react";
+import { Modal, Button, Form } from "react-bootstrap";
+
+const AdminModal = (props) => {
+  const [airmanId, setAirmanId] = useState("");
+  const [flightId, setFlightId] = useState("");
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [pt, setPT] = useState("Current");
+  const [evaluation, setEval] = useState("Current");
+  const [adls, setADLS] = useState("Current");
+  const [medical, setMedical] = useState("Current");
+
+  const createUrl = () => {
+    const key = props.optionname.toLowerCase();
+    console.log(key);
+    if (props.optionname === "Add") {
+      return `https://us-central1-cc-dashboard-afb15.cloudfunctions.net/${key}Airman/${flightId}/`;
+    } else {
+      return `https://us-central1-cc-dashboard-afb15.cloudfunctions.net/${key}Airman/${flightId}/${airmanId}`;
+    }
+  };
+
+  const getMethod = () => {
+    switch (props.optionname) {
+      case "Add":    return "POST";
+      case "Remove": return "DELETE";
+      case "Update": return "PUT";
+
+      default: return "bad-option";
+    }
+  };
```

Comment:
This method (and the other methods here) could probably be encapsulated in their own file that is imported here (we could just write them to be exported from there and so that they could have a value passed in which we would be getting from props).